examples/reference-entity-model/src/main/ml-config/security/users/wailin.json

```diff
@@ -3,10 +3,6 @@
   "description": "Wai Lin is a Systems Analyst",
   "password": "%%testPassword%%",
   "role": [
-    "hub-central-load-writer",
-    "hub-central-mapping-writer",
-    "hub-central-custom-reader",
-    "hub-central-step-runner",
-    "hub-central-downloader"
+    "hub-central-curator"
```

Comment:
Let's keep the downloader role, we need someone who's not a hc-developer that can do this for testing purposes.